src/sg/gov/tech/geekout/1.0/sample-document.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "GeekOut2020",
+  "recipient": {
+    "name": "Matthea Loo"
+  },
+  "programme": {
+    "name": "GeekOut 2020",
```

Comment:
this is useless right ?

Reply:
Currently the renderer displays the `programme.name` dynamically but I can edit the renderer to make it fixed. Would that be better?

Reply:
hum no I guess it's ok then :)